openghg/standardise/_standardise.py

```diff
@@ -839,3 +839,82 @@ def standardise_from_binary_data(
             binary_data=binary_data, metadata=metadata, file_metadata=file_metadata, **kwargs
         )
     return result
+
+
+def standardise_oned_timeseries(
+    filepath: Union[str, Path],
+    species: str,
+    source: str,
+    domain: str,
+    database: Optional[str] = None,
```

Comment:
The use of `domain` is an interesting one because in this case we're not using it to describe an explicit lat-lon range like we are for the 3D emissions and footprints.

Having a tag for the area we're considering is important but I wonder if we could:
 - add a `region` keyword as the descriptor for the area the flux data represents e.g. `region="uk"`
 - keep `domain` as optional but only use this in cases where this represents a sum over one of our named domains e.g. "europe" (not used often I'd image but perhaps useful to have)

Reply:
done

Reply:
Happy to discuss, but I don't see the `region` keyword added here if that's what we decided was the best approach?